config/crd/bases/oceanbase.oceanbase.com_obclusters.yaml

```diff
@@ -1751,6 +1751,8 @@ spec:
               serviceAccount:
                 default: default
                 type: string
+              sourceOBClusterConnection:
```

Comment:
Maybe we should run `make generate` to refresh the yaml manifests. This field had been removed ~